### PR TITLE
Add option to open door

### PR DESCRIPTION
**Pull request by Drudenhaus**

### CODE REVIEW

**Comment by Drudenhaus** [comment]
Please review this, @casey-collab

File: english/eating-walls/find-a-toilet/door/bear/run-into-room.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+You run into the room and slam the door behind you! You don't know whose hand that was and you don't want to.
```
Comment (casey-collab) [inline english/eating-walls/find-a-toilet/door/bear/run-into-room.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Drudenhaus** [comment]
I think I fixed it, @casey-collab

File: english/eating-walls/find-a-toilet/door/open-door.md [hunk 2]
```diff
@@ -4,4 +4,6 @@ You jump back as you see dark black eyes.
 
 Do you:
 
-[Stand still in terror](bear/stand-still.md)
\ No newline at end of file
+[Stand still in terror](bear/stand-still.md)
+
+[Run into the room and close the door](bear/run-into-room.md)
```
Comment (casey-collab) [inline english/eating-walls/find-a-toilet/door/open-door.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/eating-walls/find-a-toilet/door/run-in/run-into-room.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+You run into the room and slam the door behind you! You don't know whose hand that was and you don't want to.
```
Comment (casey-collab) [inline english/eating-walls/find-a-toilet/door/run-in/run-into-room.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Drudenhaus** [comment]
Right again.  I fixed the linkage to the page.  @casey-collab

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**